src/Components/Results/Needs/NeedCard.tsx

```diff
@@ -31,27 +32,52 @@ const getIconAndDefaultMessage = (
 
 const NeedCard = ({ need }: NeedsCardProps) => {
   const intl = useIntl();
+  const [infoIsOpen, setInfoIsOpen] = useState(false);
 
   const needLink = need.link.default_message;
   const translatedLink = intl.formatMessage({ id: need.link.label, defaultMessage: needLink });
 
   const needType = need.type.default_message;
   const urgentNeed = getIconAndDefaultMessage(needType, acuteConditionResultMapping);
+  const translatedNeedDesc = intl.formatMessage({
+    id: need.description.label,
+    defaultMessage: need.description.default_message,
+  });
 
   return (
-    <div className="result-program-container">
-      <div className="result-resource-more-info">
-        {urgentNeed.icon}
-        <span>
-          <ResultsTranslate translation={need.type} />
-          <strong>
-            <ResultsTranslate translation={need.name} />
-          </strong>
-        </span>
-        <a className="resource-more-info" href={translatedLink} target="_blank">
+    <div className="need-card-container">
+      <div className="header-and-button-divider">
+        <div className="result-resource-more-info">
+          {urgentNeed.icon}
+          <span>
+            <ResultsTranslate translation={need.type} />
+            <strong>
+              <ResultsTranslate translation={need.name} />
+            </strong>
+          </span>
+        </div>
+        <button
+          className={infoIsOpen ? 'more-info-btn-open' : 'more-info-btn'}
+          onClick={() => {
+            setInfoIsOpen(!infoIsOpen);
+          }}
+        >
           <FormattedMessage id="more-info" defaultMessage="More Info" />
-        </a>
+        </button>
       </div>
+      {infoIsOpen && (
+        <>
+          <article className="need-desc-paragraph">{translatedNeedDesc}</article>
```

Comment:
What do you think about using a `<p>` tag for descriptions.
```suggestion
          <p className="need-desc-paragraph"><ResultsTranslate translation={need.description} /></p>
```